client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from "react";
+import { View, Text, Image, TouchableOpacity } from "react-native";
+
+class SurveyDifficultyContainer extends Component {
+  static navigationOptions = {
+    title: "LESSONS"
+  };
+  render() {
+    return (
+      <View style={{ flex: 1, alignItems: "center", justifyContent: "center" }}>
+        <Text
+          style={{
+            paddingBottom: 40,
+            textAlign: "center",
+            fontSize: 18
+          }}
+        >
+          What's your level
+          {"\n"}
+          of financial knowledge?
+        </Text>
+
+        <View
+          style={{
+            backgroundColor: "#F2C249",
+            width: 270,
+            height: 220,
+            justifyContent: "center",
+            alignItems: "center",
+            borderRadius: 10,
+            marginBottom: 20,
+            fontFamily: "Quicksand"
+          }}
+        >
+          <View
+            style={{
+              right: 0,
+              bottom: 0,
+              backgroundColor: "transparent",
+              borderStyle: "solid",
+              borderLeftWidth: 170,
+              borderBottomWidth: 210,
+              borderBottomColor: "transparent",
+              borderLeftColor: "#edcc74",
+              transform: [{ rotate: "180deg" }],
+              position: "absolute"
+            }}
+          />
+          <Image
+            style={{
+              width: 100,
+              height: 150
+            }}
+            source={require("../../../assets/images/illustrations/PNG/Tree_oranges.png")}
+          />
+          <TouchableOpacity
+            style={{
```

Comment:
Put it into styles file

Reply:
I didn't rebase, it is already in develop